x-pack/plugins/fleet/server/services/preconfiguration.ts

```diff
@@ -226,14 +226,10 @@ export async function ensurePreconfiguredPackagesAndPolicies(
         if (wasDeleted) {
           return { created: false, deleted: preconfigurationId };
         }
-      } else if (
-        !preconfiguredAgentPolicy.is_default &&
-        !preconfiguredAgentPolicy.is_default_fleet_server
-      ) {
+      } else {
         throw new Error(
           i18n.translate('xpack.fleet.preconfiguration.missingIDError', {
-            defaultMessage:
-              '{agentPolicyName} is missing an `id` field. `id` is required, except for policies marked is_default or is_default_fleet_server.',
```

Comment:
@joshdover should we keep this check? users might have preconfigured default policies without an id that would break